### DISCUSSION TITLE
Add command r/command r plus via Azure Marketplace

### DISCUSSION THREAD

**Discussion by jayfk** [langfuse/langfuse#1800] [Ideas]
### Describe the feature or potential improvement

Getting the warning "Langfuse was not able to parse the LLM model. The LLM call will be recorded without model name. Please create an issue so we can fix your integration: https://github.com/langfuse/langfuse/issues/new/choose"

We are using Command R and Command R Plus via the Azure Marketplace: https://azuremarketplace.microsoft.com/en/marketplace/apps/cohere.cohere-command-r-plus-offer?tab=Overview

Currently via langchain:
```
chat = ChatCohere(
            base_url="<snip>",
            cohere_api_key="<snip>",
        )
```

**Comment by marcklingen** [comment]
thanks for sharing this issue, we'll try to address this in the next iteration of our langchain integration

**Comment by alex-stoica** [comment]
I'd like to add that Cohere is one of the few providers offering free API calls, making them a popular choice for building proof of concepts. For this reason, I believe it would be highly valuable to also integrate with Cohere in general (not only via LangFuse) and feature them on the Models page. However, one potential challenge is that their API does not always return the full name of the model used, which can be a bit inconvenient at times
